yateto/arch.py

```diff
@@ -79,12 +79,16 @@ def __init__(self,
     assert self.alignment % self.bytesPerReal == 0
     self.alignedReals = self.alignment // self.bytesPerReal
     self.enablePrefetch = enablePrefetch
-    
+
     self.uintTypename = 'unsigned'
     self.ulongTypename = 'unsigned long'
 
     self._tmpStackLimit = 524288
 
+    self.is_accelerator_type = False
+    if not (backend == 'cpp' or self.host_name == None):
+      self.is_accelerator_type = True
```

Comment:
`self.is_accelerator_type = backend != 'cpp' and self.host_name != None`

Reply:
done